src/Core/src/Graphics/MauiDrawable.Android.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+using Android.Graphics.Drawables.Shapes;
+using AColor = Android.Graphics.Color;
+
+namespace Microsoft.Maui.Graphics
+{
+	public class MauiDrawable : PaintDrawable
```

Comment:
This drawable only has our 3 types of brushes because we only have 3. This could be split into separate classes and each brush return it. But this is probably not worth it now as it works and is similar to what we have already.